CHANGELOG.md

```diff
@@ -28,6 +28,8 @@
 
 * **Alerting**: Reverts the behavior of `diff` and `percent_diff` to not always be absolute. Something we introduced by mistake in [6.1.0](https://github.com/grafana/grafana/commit/28eaac3a9c7082e8c496005c1cb66b4b70a4f82f). Alerting now support `diff()`, `diff_abs()`, `percent_diff()` and `percent_diff_abs()`. [#21338](https://github.com/grafana/grafana/pull/21338)
 
+* **BackendSrv**: We've changed so backendSrv uses fetch instead of using Angulars http service so the response format for .headers() changed from a function to an object.
+
```

Comment:
should this not be added below in the section `Notice about changes in backendSrv for plugin authors`

Reply:
Sure, I misunderstood what you wrote earlier.